stravalib/model.py

```diff
@@ -926,10 +1241,35 @@ class SubscriptionCallback(
     hub_verify_token: Optional[str] = None
     hub_challenge: Optional[str] = None
 
+    # TODO: error: Required dynamic aliases disallowed  [pydantic-alias]
```

Comment:
i need help with this one as well.

Reply:
i asked chatGPT and it wants an implementation like this using a static method rather than a lambda function.
```python
    @staticmethod
    def alias_generator(field_name: str) -> str:
        return field_name.replace("hub_", "hub.")
```